docs/product/explore/session-replay/web/replay-details.mdx

```diff
@@ -13,7 +13,7 @@ Every replay has a detailed view that contains the embedded video player and ric
 - **Replay Player:** Video-like reproduction of a user session created from DOM snapshots. This is where you can visualize exactly what actions the user took during a user session, and how the website behaved within this specific user’s environment, including browser, latency, settings, and so on. Most importantly, you can see which actions lead to an error and take the guesswork out of debugging.
   - By default, the Session Replay SDK is configured to replace all text nodes and user input with asterisks (\*), and to remove media elements. See all privacy configuration options [here](/platforms/javascript/session-replay/privacy/).
 
-* **Breadcrumbs:** The replay breadcrumbs show when key user interactions took place, specifically: user clicks (including rage and dead clicks), user inputs, [page navigations, page loads](/product/performance/transaction-summary/#what-is-a-transaction), as well as [custom breadcrumbs](/product/sentry-basics/integrate-backend/configuration-options/#breadcrumbs) set by the organization and the [Largest Contentful Paint (LCP)](/product/insights/web-vitals/web-vitals-concepts/#largest-contentful-paint-lcp) web vital. User clicks (including rage and dead clicks), are augmented with surrounding HTML source code to help users understand which event was targeted. These breadcrumbs are synced with the replay player and will auto-scroll as the video plays.
+* **Breadcrumbs:** The replay breadcrumbs show when key user interactions took place, specifically: user clicks (including rage and dead clicks), user inputs, [page navigations, page loads](/product/performance/transaction-summary/#what-is-a-transaction), as well as [custom breadcrumbs](/product/sentry-basics/integrate-backend/configuration-options/#breadcrumbs) set by the organization and [web vitals](/product/insights/web-vitals/). User clicks (including rage and dead clicks), are augmented with surrounding HTML source code to help users understand which event was targeted. Web vitals are augmented with the selector to help users identify which elements are affecting their web vital scores. These breadcrumbs are synced with the replay player and will auto-scroll as the video plays.
```

Comment:
> Web vitals are augmented with the selector to help users identify which elements are affecting their web vital scores

Should we move this sentence to be immediately following the web vitals link? That would seem to make sense